c2corg_ui/templates/helpers/view.html

```diff
@@ -243,7 +243,7 @@ <h3 class="heading show-phone" ng-click="mainCtrl.animateHeaderIcon($event)" dat
     </section>
 </%def>
 
-<%def name="show_associated_xreports(document)">\
+<%def name="show_associated_xreports(document, type='xreports')">\
```

Comment:
Where are you calling the function with the optional parameter set? Don't you rather want to initialize a normal variable below?

```
<%
type = 'xreports'
xreports = document['associations']['xreports']
...
```